recipe/patches/0010-Adopt-system-libprotobuf.patch

```diff
@@ -0,0 +1,2654 @@
+From b67c290f7a379ad8a9d779754b10b2602fbb99c7 Mon Sep 17 00:00:00 2001
+From: "Uwe L. Korn" <uwe.korn@quantco.com>
+Date: Thu, 27 Jun 2024 17:08:56 +0200
+Subject: [PATCH 10/11] Adopt system libprotobuf
+
+---
+ third_party/systemlibs/protobuf/BUILD         |    4 +-
+ third_party/systemlibs/protobuf/MODULE.bazel  |   12 +-
+ .../systemlibs/protobuf/java/core/BUILD       |    6 +
+ .../protobuf/maven_protobuf_install.json      | 2572 +++++++++++++++++
```

Comment:
Could you leave a comment how this file is supposed to be generated? Also, if easily automatable, perhaps it would be easier to download/generate this json on the fly in the build script, rather than copy it in via a patch.

Reply:
I will try to switch to generation in the build script before I merge.